### PR TITLE
Added findIndex and findLastIndex functions

### PR DESCRIPTION
**Pull request by denisjuergen**
Like mentioned in the commit messages, I've added core helper functions needed by those functions as well.

### CODE REVIEW

File: underscore-updates.js [hunk 1]
```diff
@@ -37,6 +37,31 @@ var cb = function(value, context, argCount) {
...
+    for (; index >= 0 && index < length; index += dir) {
+      if (predicate(array[index], index, array)) return index;
+    }
+    return -1;
+  };
+};
+
+// Returns the first index on an array-like that passes a predicate test
+_.findIndex = createPredicateIndexFinder(1);
+_.findLastIndex = createPredicateIndexFinder(-1);
```
Comment (bradvogel) [inline underscore-updates.js:63]: Please add these to the readme.

File: underscore [hunk 2]
```diff
@@ -0,0 +1 @@
+Subproject commit 101c10a60019fe870d21868835f65c25d64968fc
```
Comment (wearhere) [inline underscore]: Hm why's a submodule necessary? Given that the package [implies](https://github.com/mixmaxhq/meteor-underscore-updates/blob/master/package.js#L12) that Underscore's already part of the project.

**Comment by denisjuergen** [comment]
@bradvogel Forgot about that. Will do.
@wearhere I guess you're right.

Do you see any problem in rebasing my local repository to remove the commit where I added the submodule? That way I'd also just amend the commit where I've updated the README.md. Please note that I'd have to force push the commit. Just letting you know in case anyone is tracking any commits from my repo.

**Comment by wearhere** [comment]
Go ahead and rebase/force-push @denisjuergen! Thanks.

**Comment by denisjuergen** [comment]
Done. Looks like everything is ok now. Otherwise, let me know!

**Comment by wearhere** [comment]
Looks great @denisjuergen! Thanks for the PR. Always nice to see someone who appreciates a clean Git history too :)

**Comment by wearhere** [comment]
Version bumped to 0.1.3 and published too.

**Comment by denisjuergen** [comment]
No problem @wearhere ! Yeah, I have a real problem with my git repositories log history.. haha
Making it a dependency on my project tomorrow morning. Thanks!

**Pull request merged**